bundles/AdminBundle/Controller/Admin/Document/FolderController.php

```diff
@@ -42,6 +42,10 @@ public function getDataByIdAction(Request $request)
     {
         $folder = Document\Folder::getById($request->get('id'));
 
+        if (!$folder) {
+            throw $this->createNotFoundException();
```

Comment:
maybe add a message

Reply:
Default message is "Not found" and its a 404. Not enough?

Reply:
for now I would leave it as it is

Reply:
I set it, so the user know at least the type